Angular/angular.md

```diff
@@ -0,0 +1,45 @@
+
+# Angular
+
+## Motivation
+
+- Angular is a popular Javascript Framework that is being used to build Applications for Web, Desktop and Mobile Devices. 
+- Launched by Google in 2010 as AngularJS, the subsequent versions of AngularJS are simply called Angular and involves the usage of Typescript which is a superset of Javascript. 
+- Typescript is compatible with Javascript and compiles to Javascript which is rendered on the Browser.
+- Angular is similar to React.js and Vue which are being used for creating Front-End User Interfaces.
+- What makes Angular quite intutive and easy to learn is that, Angular uses HTML to define the User-Interface and hence we don't need to write the Front-End Interface in Javascript. 
+- HTML is a Markup language and follows a well-defined layout where we can bind the data using Angular and readily render an interface with minimal effort.
```

Comment:
Please condense this into a one sentence reason why someone who, say, already learned React should bother with Angular.